migrations/sql/V1.9__Add_Mine_Type_and_Mine_Tenure_Type.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE mine_tenure_type (
+  mine_tenure_type_id   smallint PRIMARY KEY NOT NULL,
```

Comment:
Can you add indexes to the foreign/join columns. AND/OR to the columns you know there will be a lot of querying/sorting on